src/inputs/ObjectSearch.tsx

```diff
@@ -44,81 +49,103 @@ class ObjectSearch extends Component<IObjectSearchProps> {
     return this.props.fieldConfig as IFieldConfigObjectSearchCreate;
   }
 
+  private get noSearch () {
```

Comment:
Great clean up !
One thing that might be better though is keeping our "state naming" the same as we do in our other codebase.
for example:
```ts
  private get noSearch () { ... }
  private get noOptions () { ... }
```

Would one of these two be better and more descriptive ?
1.
```ts
  private get hasNoSearch () { ... }
  private get hasNoOptions () { ... }
```
2.
```ts
  private get isNoSearch () { ... }
  private get isNoOptions () { ... }
```

Reply:
Yeah I thought about that but `isNoSearch` made no sense whatsoever. `hasNoSearch` looks a lot better.